common/src/main/scala/uk/ac/wellcome/models/Item.scala

```diff
@@ -7,7 +7,6 @@ case class Item(
   sourceIdentifier: SourceIdentifier,
   identifiers: List[SourceIdentifier] = Nil,
   locations: List[Location] = List(),
-  visible: Boolean = true
-) extends Identifiable {
-  @JsonProperty("type") val ontologyType: String = "Item"
-}
+  visible: Boolean = true,
+  @JsonProperty("type") ontologyType: String = "Item"
```

Comment:
Had to pull this into the parameter list cause circe relies on shapeless instead of reflection, which doesn't give visibility on members in the class body. 
I did prefer them being constant, but, meh 😐

Reply:
Yeah, except they aren’t sometimes, so we'd have done this anyway 😕